test/ectest.c

```diff
@@ -2361,10 +2364,8 @@ static int custom_generator_test(int id)
         goto err;
 
     /* expected byte length of encoded points */
-    bsize = (EC_GROUP_get_field_type(group) == NID_X9_62_prime_field) ?
-             BN_num_bytes(EC_GROUP_get0_field(group)) :
-             (EC_GROUP_get_degree(group) + 7) / 8;
-    bsize = 2 * bsize + 1;
+    bsize = (EC_GROUP_get_degree(group) + 7) / 8;
```

Comment:
Can you explain this change?

Reply:
We did this in 1.1.1 already: https://github.com/openssl/openssl/pull/12433

Basically `EC_GROUP_get_degree()` (which name makes sense for the binary curves but not for prime curves) already does "the right thing" for prime curves, making it unnecessary to have that distinction altogether.